package.json

```diff
@@ -15,51 +15,51 @@
     "flux": "3.1.3",
     "font-awesome": "4.7.0",
     "highlight.js": "9.12.0",
-    "hoist-non-react-statics": "2.5.0",
+    "hoist-non-react-statics": "2.5.5",
     "html-to-react": "1.3.3",
     "inobounce": "0.1.6",
     "intl": "1.2.5",
     "jasny-bootstrap": "3.1.3",
     "jquery": "3.3.1",
     "katex": "0.9.0",
     "key-mirror": "1.0.1",
-    "localforage": "1.7.1",
+    "localforage": "1.7.2",
     "localforage-observable": "1.4.0",
     "marked": "github:mattermost/marked#ed33baecd7d7fa97d479ba22dde9d226b083d67d",
     "mattermost-redux": "github:mattermost/mattermost-redux#6aee7c45bcde7d2079ee5b24f70e7e8d071ae0a6",
-    "moment-timezone": "0.5.17",
+    "moment-timezone": "0.5.20",
     "pdfjs-dist": "2.0.489",
     "perfect-scrollbar": "0.8.1",
-    "prop-types": "15.6.1",
-    "react": "16.3.2",
+    "prop-types": "15.6.2",
+    "react": "16.4.1",
     "react-addons-pure-render-mixin": "15.6.2",
     "react-bootstrap": "0.32.1",
     "react-color": "2.14.1",
     "react-contextmenu": "2.9.2",
     "react-custom-scrollbars": "4.2.1",
-    "react-dom": "16.3.2",
+    "react-dom": "16.4.1",
     "react-intl": "2.4.0",
     "react-redux": "5.0.7",
-    "react-router-dom": "4.2.2",
+    "react-router-dom": "4.3.1",
     "react-select": "1.2.1",
     "redux": "4.0.0",
     "redux-batched-actions": "0.3.0",
     "redux-persist": "4.9.1",
-    "redux-persist-transform-filter": "0.0.16",
+    "redux-persist-transform-filter": "0.0.18",
     "reselect": "3.0.1",
     "smoothscroll-polyfill": "0.4.3",
     "superagent": "3.8.3",
-    "twemoji": "2.5.1",
+    "twemoji": "11.0.0",
```

Comment:
This huge change in the version number is to align with Unicode supported version. But looks like a no-breaking change.